src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,298 @@
+
+        //Just remove the attributes not the metadata
+        jedis.del((session.getId() + REDIS_ATTRIBUTES_KEY).getBytes());
+
+        redisConnectionPool.returnResourceObject(jedis);
+    }
+
+    @Override
+    public void processExpires() {
+       //We need to implement the expire process
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+    // END Sessions
+    //------------------------------------------------------------------------------------------------------------------
+
+
+    //------------------------------------------------------------------------------------------------------------------
+    // START Utils
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * Get session from metadata and attributes
+     * @param metadata
+     * @param attributes
+     * @return
+     */
+    private StandardSession getSession(Hashtable<String, Object> metadata, Hashtable<String, Object> attributes){
+        StandardSession standardSession = (StandardSession) createEmptySession();
+        standardSession.setValid(Boolean.valueOf((String)metadata.get(METADATA_VALID)));
+        SimpleDateFormat format = new SimpleDateFormat(ISO_DATE_FORMAT);
+        try {
+            standardSession.setCreationTime(format.parse(((String) metadata.get(METADATA_CREATION_TIME))).getTime());
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        standardSession.setMaxInactiveInterval(Integer.valueOf((String) metadata.get(METADATA_MAX_INACTIVE_INTERVAL)));
+        return standardSession;
+    }
+
+
+    /**
+     * Get metadata from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getMetadata(Session session){
+        Hashtable<String, Object> metadata = new Hashtable<>();
+        metadata.put(METADATA_VALID, String.valueOf(session.isValid()));
+        metadata.put(METADATA_CREATION_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(session.getCreationTime()));
+        metadata.put(METADATA_LAST_ACCESS_TIME, DateFormatUtils.ISO_DATE_FORMAT.format(session.getLastAccessedTime()));
+        metadata.put(METADATA_MAX_INACTIVE_INTERVAL, String.valueOf(session.getMaxInactiveInterval()));
+        return metadata;
+    }
+
+    /**
+     * Get attributes from a Session
+     * @param session
+     * @return
+     */
+    private Hashtable<String, Object> getAttributes(Session session){
+        StandardSession standardSession = (StandardSession)session;
+        Hashtable<String, Object> attributes = new Hashtable();
+        for (Enumeration<String> enumerator = standardSession.getAttributeNames(); enumerator.hasMoreElements();) {
+            String key = enumerator.nextElement();
+            attributes.put(key, standardSession.getAttribute(key));
+        }
+        return attributes;
```

Comment:
Why are you building it out here instead of getting an access at it before? You pass this `Hashtable` into `getSession`.